components/Form/__tests__/MultiStepForm.test.js

```diff
@@ -169,69 +169,73 @@ describe('MultiStepForm', () => {
     const originalConsoleError = console.error;
     console.error = jest.fn();
 
-    const wrapper = mount(<MultiStepForm {...requiredProps} />);
-    expect(wrapper.find('form').exists()).toBe(true);
-    expect(wrapper.find('input').exists()).toBe(true);
-    expect(wrapper.find('button').text()).toContain('Next');
+    const { container } = render(<MultiStepForm {...requiredProps} />);
+    expect(container.querySelector('form')).toBeInTheDocument();
```

Comment:
`toBeInTheDocument` is being deprecated. I'd prefer to see `toBeDefined();` or `not.isNull();`

Reply:
sure